charts/sigstore-prober/templates/namespace.yaml

```diff
@@ -0,0 +1,6 @@
+{{- if .Values.namespace.create }}
```

Comment:
i don't think we need to set this because we can use the helm cli to create or point to the namespace we want to use.

but also i don't have a strong opinion on this

Reply:
This would follow the pattern implemented by other charts in the repository. This parameter was designed to support the scaffold chart

Reply:
ahh, i forgot those 🤦